corehq/warehouse/migrations/0032_auto_20190917_1542.py

```diff
@@ -0,0 +1,46 @@
+from django.db import migrations, models
+
+TABLE_NAME = 'warehouse_synclogstagingtable'
+INDEX_NAME = 'warehouse_s_user_id_8bcfbf_idx'
```

Comment:
just curious about the `_s_` in the name?

Reply:
this is autogenerated by django. likely just a truncation `warehouse_synclogstagingtable`

Reply:
Thanks. I thought we write such migrations manually and hence i have always tried to come up with short unique name myself.